IDE/apple-universal/README.md

```diff
@@ -0,0 +1,79 @@
+# Overview
+This example shows how to build a wolfSSL static library for Apple targets on all architectures using GNU autotools/`configure` and demonstrates how to create a [universal binary framework]() suitable for use in an Xcode project. It also provides a demo Xcode project using the wolfSSL framework in a simple multiplatform app.
```

Comment:
Please document what version of Xcode this is tested with... My 13.2.1 could not open the wolfssl-multiplatform.xcodeproj (too old).